queries.sql

```diff
@@ -1,3 +1,23 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE 'mon%';
+
+SELECT name FROM animals 
+WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+
+SELECT name FROM animals 
+WHERE neutered=TRUE AND escape_attempts<3;
+
+SELECT date_of_birth FROM animals 
+WHERE name='Agumon' OR  name='Pikachu';
+
+SELECT name, escape_attempts                                                                        FROM animals                                                                                                     WHERE weight_kg>10.5;
+
+SELECT * FROM animals                                                                                                                                                                WHERE neutered=TRUE;
+
```

Comment:
- [ ] These two quires are not completed. Please work on these queries. 
- It can list name and escape attempts of animals that weigh more than 10.5kg
- It can find all animals that are neutered.